packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -133,10 +143,21 @@ const styles = StyleSheet.create({
         borderTopWidth: 1,
     },
 
+    // This element is centered within the titlebar, despite the presence of
+    // the close button, because there's an additional right-hand spacer
+    // element that grows at the same rate.
     titleAndSubtitle: {
-        display: "flex",
-        flexDirection: "column",
-        justifyContent: "center",
+        flex: "0 0 auto",
         textAlign: "center",
+        paddingLeft: 16,
+        paddingRight: 16,
+    },
+
+    closeButton: {
+        flex: "1 0 0",
```

Comment:
The positioning of the close button is off.

Reply:
Not sure I understand this one—is it fixed with the new bounding box updates?